ledger-core/insolar/payload/validation.go

```diff
@@ -5,175 +5,6 @@
 
 package payload
 
-import (
-	"github.com/insolar/assured-ledger/ledger-core/vanilla/throw"
-)
-
-type Validate interface {
+type Validatable interface {
```

Comment:
go way it would be Valitor

Reply:
but this one is nice too

Reply:
Yes, I've though about it, but on other hand `Validator` is something that validates what's provided, but `Validatable` shows that it can be validated. It's dilemma that i've faced.